src/test/java/teammates/test/cases/common/TimeHelperTest.java

```diff
@@ -82,6 +82,40 @@ public void testCombineDateTime() {
     }
     
     @Test
+    public void testIsWithinHour() {
+        Calendar timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        
+        ______TS("Time within past hour");
+        
+        timeCalendar.add(Calendar.MINUTE, -10);
+        assertTrue(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                   Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+
+        ______TS("End time not within past hour");
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, -70);
+        assertFalse(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                    Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, 10);
+        assertFalse(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                    Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
```

Comment:
`Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime())` repeats a lot. May be abstract into something like `TimeHelper.now()`?
Another thing: Why not get it once and reuse for the entire test? Getting it multiple times means you are using test data that are different and the exact value is unknown. 
The test cases could go like this:
```
deadline = getNow()
current = deadline
test with deadline, current,

current = current +1 hour
test with deadline, current

current = current + 1 millisecond
test with deadline, current

```